### PR TITLE
Blog post for position paper

### PR DESCRIPTION
**Pull request by c-martinez**
Once post is published, we can ask others to cross-post & start sharing the paper.

### CODE REVIEW

File: _posts/2020-09-22-RS-first-class-citizen.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+*Image credit: [Clive Warneford](https://commons.wikimedia.org/wiki/File:Class_159_First_Class_Seating.jpg) / [CC BY-SA](https://creativecommons.org/licenses/by-sa/2.0)*
+
+
+TL;DR -- Read [our recommendations](https://doi.org/10.5281/zenodo.4015242) to make research software a first-class citizen.
+
+<!--break-->
+
+Antoni van Leeuwenhoek is considered the father of microbiology. His discovery of microbes, using the lenses he made himself, created an entire new field of research. He was at the same time a researcher and a tool maker: his research would have not been possible without the tools he built. Leeuwenhoek was well known for both his discoveries in microbiology as well as the unmatched quality of his lenses.
+
+Four centuries after Leeuwenhoek, research tools have only gained importance. However, in contrast with the past, a new type of tool gained critical importance: research software. The 2020 COVID-19 pandemic has brought up to the public eye how [important research software is] (https://www.software.ac.uk/blog/2020-05-07-better-software-better-covid-19-research).
```
Comment (nielsdrost) [inline _posts/2020-09-22-RS-first-class-citizen.md:16]:
```suggestion
Four centuries after Leeuwenhoek, research tools have only gained importance. Recently, a new type of tool gained critical importance: research software. The 2020 COVID-19 pandemic has brought up to the public eye how [important research software is] (https://www.software.ac.uk/blog/2020-05-07-better-software-better-covid-19-research).

```

Comment (nielsdrost) [reply _posts/2020-09-22-RS-first-class-citizen.md:16]: I'm not sure "However" makes sense in this context. Replaced with something a bit more neutral.

File: _posts/2020-09-22-RS-first-class-citizen.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+layout: post
+image: '/img/Class_159_First_Class_Seating.jpg'
+title: "How to make research software a first-class citizen in the Netherlands?"
```
Comment (nielsdrost) [inline _posts/2020-09-22-RS-first-class-citizen.md:4]: Why the questionmark?

**Review by nielsdrost — APPROVED** [review]
Nice! I made a few suggestions. Feel free to ignore ;-)

File: _posts/2020-09-22-RS-first-class-citizen.md (hunk 2 above)
Comment (c-martinez) [reply _posts/2020-09-22-RS-first-class-citizen.md:4]: I guess it is a question 'the reader' is asking to himself 
Q: How do I make the world better?
A: By following these recommendations.

But we can remove the question mark if you feel it doesn't make sense.

**Comment by c-martinez** [comment]
@nielsdrost -- I'll merge this now, and we can change the title later if you feel it makes more sense without the question mark.

File: _posts/2020-09-22-RS-first-class-citizen.md (hunk 2 above)
Comment (nielsdrost) [reply _posts/2020-09-22-RS-first-class-citizen.md:4]: No problem, we can also leave it like this? ;-)

**Pull request merged**